packages/dlog/src/dlog.ts

```diff
@@ -169,14 +169,26 @@ const makeDlog = (d: Debugger, opts?: DLogOpts): DLogger => {
                 newArgs.push(c)
             } else if (typeof c === 'object' && c !== null) {
                 if (c instanceof Error) {
-                    isSingleLineLogsMode ? fmt.push('%o\n') : fmt.push('%O\n')
+                    if (isSingleLineLogsMode) {
+                        fmt.push('%o\n')
+                    } else {
+                        fmt.push('%O\n')
+                    }
                     tailArgs.push(c)
                 } else {
-                    isSingleLineLogsMode ? fmt.push('%o\n') : fmt.push('%O\n')
+                    if (isSingleLineLogsMode) {
+                        fmt.push('%o\n')
+                    } else {
+                        fmt.push('%O\n')
+                    }
                     newArgs.push(cloneAndFormat(c, { shortenHex: true }))
                 }
             } else {
-                isSingleLineLogsMode ? fmt.push('%o ') : fmt.push('%O ')
+                if (isSingleLineLogsMode) {
+                    fmt.push('%o ')
+                } else {
+                    fmt.push('%O ')
+                }
```

Comment:
why? did ternaries change?

Reply:
eslint added a new default rule about unused expressions. i.e ternary to run side effects. I disabled it on harmony, I will update and disable it here too

Reply:
honestly - that's pretty nice. you can leave this.